SharedPackages/BrowserServicesKit/Sources/Bookmarks/FaviconsFetcher/BookmarksFaviconsFetcher.swift

```diff
@@ -182,7 +182,8 @@ public final class BookmarksFaviconsFetcher {
      *
      * This function cancels any pending fetch operation and schedules a new operation.
      */
-    public func startFetching() {
+    public func startFetching() async {
```

Comment:
Making this function async so that it can await the faviconStore, allowing to request the store on the main thread.